okio/src/zlibMain/kotlin/okio/ZipFileSystem.kt

```diff
@@ -24,7 +24,7 @@ import okio.internal.readLocalHeader
 import okio.internal.skipLocalHeader
 
 /**
- * Read only access to a [zip file][zip_format] and common [extra fields][extra_fields].
+ * Read only access to a [zip file](zip_format) and common [extra fields](extra_fields).
  *
  * [zip_format]: https://pkware.cachefly.net/webdocs/APPNOTE/APPNOTE_6.2.0.txt
  * [extra_fields]: https://opensource.apple.com/source/zip/zip-6/unzip/unzip/proginfo/extra.fld
```

Comment:
Doesn't look like it works for me. The links are there but nothing happens as I click on them.

![image](https://github.com/square/okio/assets/1767669/1063d4d7-5247-4eb4-a202-8afbc122e7dc)

I have to inline them to see something working.

![image](https://github.com/square/okio/assets/1767669/032be44f-a8c7-405f-85c5-824ba8265344)


```suggestion
 * Read only access to a [zip file](https://pkware.cachefly.net/webdocs/APPNOTE/APPNOTE_6.2.0.txt)
 * and common [extra fields](https://opensource.apple.com/source/zip/zip-6/unzip/unzip/proginfo/extra.fld).
```